server/channels/api4/system.go

```diff
@@ -657,16 +682,26 @@ func getRedirectLocation(c *Context, w http.ResponseWriter, r *http.Request) {
 	// If the location length is > 2100, we can probably ignore. Fixes https://mattermost.atlassian.net/browse/MM-54219
 	if len(location) > RedirectLocationMaximumLength {
 		// Treating as a "failure". Cache failures to prevent retries.
-		redirectLocationDataCache.SetWithExpiry(url, "", RedirectLocationCacheExpiry)
+		cacheErr := redirectLocationDataCache.SetWithExpiry(url, "", RedirectLocationCacheExpiry)
+		if cacheErr != nil {
+			c.Logger.Warn("failed to set cache for url", mlog.Err(err))
+		}
 		// Always return a success status and a JSON string to limit information returned to client.
-		w.Write([]byte(model.MapToJSON(m)))
+		if _, err = w.Write([]byte(model.MapToJSON(m))); err != nil {
+			c.Logger.Warn("Error while writing response", mlog.Err(err))
+		}
 		return
 	}
 
-	redirectLocationDataCache.SetWithExpiry(url, location, RedirectLocationCacheExpiry)
+	cacheErr := redirectLocationDataCache.SetWithExpiry(url, location, RedirectLocationCacheExpiry)
+	if cacheErr != nil {
+		c.Logger.Warn("failed to set cache for url", mlog.Err(err))
```

Comment:
Can you please capitalize the first word here and in the other log messages?
```suggestion
		c.Logger.Warn("Failed to set cache for url", mlog.Err(err))
```

Reply:
Fixed!! and log the URL here as well!